hilti/runtime/include/type-info.h

```diff
@@ -691,7 +691,7 @@ class MapIterator {
 private:
     const TypeInfo* _ktype;
     const TypeInfo* _vtype;
-    const Accessor _accessor;
+    Accessor* _accessor;
```

Comment:
Should this really be a pointer? `Accessor` is a pair here, no?

Reply:
Accessor is a function `const Value&` -> `std::pair<const void*, const void*` so this is fine. I changed this to a function pointer type as well like you suggested above.